2020-09-05 11:12:01,268 ERROR 7480 nodejs.ClusterClientNoResponseError: client no response in 890041ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_cluster-client@3.0.1@cluster-client\lib\leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 7480
hostname: LAPTOP-7S7JCR14

2020-09-05 11:18:12,755 ERROR 14592 [-/::1/-/10ms POST /getadmin] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'en' in 'where clause'
    at Query.Sequence._packetToError (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Pool.js:199:23)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:12:10)
    at Pool.ret [as query] (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:56:34)
    at RDSClient.proto._query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:65:19)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:54:5
    sql: select * from user2 where id=en-us
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'en' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "select * from user2 where id=en-us"
headers: {"Access-Control-Allow-Origin":"*","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 14592
hostname: LAPTOP-7S7JCR14

2020-09-05 11:18:12,763 ERROR 14592 [-/::1/-/17ms POST /getadmin] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'en' in 'where clause'
    at Query.Sequence._packetToError (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Pool.js:199:23)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:12:10)
    at Pool.ret [as query] (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:56:34)
    at RDSClient.proto._query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:65:19)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:54:5
    sql: select * from user2 where id=en-us
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'en' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "select * from user2 where id=en-us"
headers: {"Access-Control-Allow-Origin":"*","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 14592
hostname: LAPTOP-7S7JCR14

2020-09-05 12:22:08,351 ERROR 7480 nodejs.ClusterClientNoResponseError: client no response in 3224588ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_cluster-client@3.0.1@cluster-client\lib\leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 7480
hostname: LAPTOP-7S7JCR14

2020-09-05 12:23:43,004 ERROR 14592 [-/::1/-/5ms POST /getadmin] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'en' in 'where clause'
    at Query.Sequence._packetToError (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Pool.js:199:23)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:12:10)
    at Pool.ret [as query] (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:56:34)
    at RDSClient.proto._query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:65:19)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:54:5
    sql: select * from user2 where id=en-us
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'en' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "select * from user2 where id=en-us"
headers: {"Access-Control-Allow-Origin":"*","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 14592
hostname: LAPTOP-7S7JCR14

2020-09-05 12:23:43,010 ERROR 14592 [-/::1/-/8ms POST /getadmin] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'en' in 'where clause'
    at Query.Sequence._packetToError (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Pool.js:199:23)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:12:10)
    at Pool.ret [as query] (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:56:34)
    at RDSClient.proto._query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:65:19)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:54:5
    sql: select * from user2 where id=en-us
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'en' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "select * from user2 where id=en-us"
headers: {"Access-Control-Allow-Origin":"*","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 14592
hostname: LAPTOP-7S7JCR14

2020-09-05 12:24:02,257 ERROR 14592 [-/::1/-/2ms POST /getadmin] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'en' in 'where clause'
    at Query.Sequence._packetToError (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Pool.js:199:23)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:12:10)
    at Pool.ret [as query] (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:56:34)
    at RDSClient.proto._query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:65:19)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:54:5
    sql: select * from user2 where id=en-us
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'en' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "select * from user2 where id=en-us"
headers: {"Access-Control-Allow-Origin":"*","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 14592
hostname: LAPTOP-7S7JCR14

2020-09-05 12:24:02,264 ERROR 14592 [-/::1/-/4ms POST /getadmin] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'en' in 'where clause'
    at Query.Sequence._packetToError (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Pool.js:199:23)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:12:10)
    at Pool.ret [as query] (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:56:34)
    at RDSClient.proto._query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:65:19)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:54:5
    sql: select * from user2 where id=en-us
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'en' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "select * from user2 where id=en-us"
headers: {"Access-Control-Allow-Origin":"*","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 14592
hostname: LAPTOP-7S7JCR14

2020-09-05 12:24:17,040 ERROR 14592 [-/::1/-/2ms POST /getadmin] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'en' in 'where clause'
    at Query.Sequence._packetToError (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Pool.js:199:23)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:12:10)
    at Pool.ret [as query] (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:56:34)
    at RDSClient.proto._query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:65:19)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:54:5
    sql: select * from user2 where id=en-us
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'en' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "select * from user2 where id=en-us"
headers: {"Access-Control-Allow-Origin":"*","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 14592
hostname: LAPTOP-7S7JCR14

2020-09-05 12:24:20,420 ERROR 14592 [-/::1/-/1ms POST /getadmin] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'en' in 'where clause'
    at Query.Sequence._packetToError (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Pool.js:199:23)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:12:10)
    at Pool.ret [as query] (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:56:34)
    at RDSClient.proto._query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:65:19)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:54:5
    sql: select * from user2 where id=en-us
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'en' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "select * from user2 where id=en-us"
headers: {"Access-Control-Allow-Origin":"*","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 14592
hostname: LAPTOP-7S7JCR14

2020-09-05 12:24:26,070 ERROR 14592 [-/::1/-/3ms POST /getadmin] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'en' in 'where clause'
    at Query.Sequence._packetToError (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Pool.js:199:23)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:12:10)
    at Pool.ret [as query] (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:56:34)
    at RDSClient.proto._query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:65:19)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:54:5
    sql: select * from user2 where id=en-us
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'en' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "select * from user2 where id=en-us"
headers: {"Access-Control-Allow-Origin":"*","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 14592
hostname: LAPTOP-7S7JCR14

2020-09-05 12:27:02,623 ERROR 14592 [-/::1/-/4ms POST /getadmin] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'en' in 'where clause'
    at Query.Sequence._packetToError (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Pool.js:199:23)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:12:10)
    at Pool.ret [as query] (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:56:34)
    at RDSClient.proto._query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:65:19)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:54:5
    sql: select * from user2 where id=en-us
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'en' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "select * from user2 where id=en-us"
headers: {"Access-Control-Allow-Origin":"*","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 14592
hostname: LAPTOP-7S7JCR14

2020-09-05 12:27:02,630 ERROR 14592 [-/::1/-/9ms POST /getadmin] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'en' in 'where clause'
    at Query.Sequence._packetToError (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Pool.js:199:23)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:12:10)
    at Pool.ret [as query] (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:56:34)
    at RDSClient.proto._query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:65:19)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:54:5
    sql: select * from user2 where id=en-us
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'en' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "select * from user2 where id=en-us"
headers: {"Access-Control-Allow-Origin":"*","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 14592
hostname: LAPTOP-7S7JCR14

2020-09-05 12:27:14,623 ERROR 14592 [-/::1/-/3ms POST /getadmin] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'en' in 'where clause'
    at Query.Sequence._packetToError (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Pool.js:199:23)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:12:10)
    at Pool.ret [as query] (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:56:34)
    at RDSClient.proto._query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:65:19)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:54:5
    sql: select * from user2 where id=en-us
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'en' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "select * from user2 where id=en-us"
headers: {"Access-Control-Allow-Origin":"*","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 14592
hostname: LAPTOP-7S7JCR14

2020-09-05 12:27:14,633 ERROR 14592 [-/::1/-/12ms POST /getadmin] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'en' in 'where clause'
    at Query.Sequence._packetToError (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Pool.js:199:23)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:12:10)
    at Pool.ret [as query] (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:56:34)
    at RDSClient.proto._query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:65:19)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:54:5
    sql: select * from user2 where id=en-us
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'en' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "select * from user2 where id=en-us"
headers: {"Access-Control-Allow-Origin":"*","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 14592
hostname: LAPTOP-7S7JCR14

2020-09-05 12:28:20,209 ERROR 14592 [-/::1/-/4ms POST /getadmin] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'en' in 'where clause'
    at Query.Sequence._packetToError (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Pool.js:199:23)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:12:10)
    at Pool.ret [as query] (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:56:34)
    at RDSClient.proto._query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:65:19)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:54:5
    sql: select * from user2 where id=en-us
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'en' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "select * from user2 where id=en-us"
headers: {"Access-Control-Allow-Origin":"*","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 14592
hostname: LAPTOP-7S7JCR14

2020-09-05 12:28:20,215 ERROR 14592 [-/::1/-/8ms POST /getadmin] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'en' in 'where clause'
    at Query.Sequence._packetToError (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Pool.js:199:23)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:12:10)
    at Pool.ret [as query] (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:56:34)
    at RDSClient.proto._query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:65:19)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:54:5
    sql: select * from user2 where id=en-us
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'en' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "select * from user2 where id=en-us"
headers: {"Access-Control-Allow-Origin":"*","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 14592
hostname: LAPTOP-7S7JCR14

2020-09-05 12:35:26,622 ERROR 14592 [-/::1/-/2ms POST /getadmin] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'en' in 'where clause'
    at Query.Sequence._packetToError (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Pool.js:199:23)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:12:10)
    at Pool.ret [as query] (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:56:34)
    at RDSClient.proto._query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:65:19)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:54:5
    sql: select * from user2 where id=en-us
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'en' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "select * from user2 where id=en-us"
headers: {"Access-Control-Allow-Origin":"*","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 14592
hostname: LAPTOP-7S7JCR14

2020-09-05 12:35:37,418 ERROR 14592 [-/::1/-/2ms POST /getadmin] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'en' in 'where clause'
    at Query.Sequence._packetToError (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Pool.js:199:23)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:12:10)
    at Pool.ret [as query] (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:56:34)
    at RDSClient.proto._query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:65:19)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:54:5
    sql: select * from user2 where id=en-us
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'en' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "select * from user2 where id=en-us"
headers: {"Access-Control-Allow-Origin":"*","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 14592
hostname: LAPTOP-7S7JCR14

2020-09-05 12:36:00,426 ERROR 14592 [-/::1/-/3ms POST /getadmin] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'en' in 'where clause'
    at Query.Sequence._packetToError (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Pool.js:199:23)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:12:10)
    at Pool.ret [as query] (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:56:34)
    at RDSClient.proto._query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:65:19)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:54:5
    sql: select * from user2 where id=en-us
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'en' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "select * from user2 where id=en-us"
headers: {"Access-Control-Allow-Origin":"*","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 14592
hostname: LAPTOP-7S7JCR14

2020-09-05 12:58:22,666 ERROR 14592 [-/::1/-/19ms POST /zhongjian] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1
    at Query.Sequence._packetToError (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_mysql@2.18.1@mysql\lib\Pool.js:199:23)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:12:10)
    at Pool.ret [as query] (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_pify@2.3.0@pify\index.js:56:34)
    at RDSClient.proto._query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_ali-rds@3.4.0@ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:65:19)
    at D:\hqyj\Webpack\vuew\mstx_server\node_modules\_co@4.6.0@co\index.js:54:5
    sql: select *  from food where id in()
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1"
sqlState: "42000"
index: 0
sql: "select *  from food where id in()"
headers: {"Access-Control-Allow-Origin":"*","vary":"Origin"}
name: "ER_PARSE_ERRORError"
pid: 14592
hostname: LAPTOP-7S7JCR14

